string direction
---
sensor_msgs/Image answer_view
int32 camera_angle
